go.mod

```diff
@@ -8,10 +8,8 @@ require (
 	github.com/asaskevich/govalidator v0.0.0-20190424111038-f61b66f89f4a
 	github.com/cavaliercoder/grab v2.0.0+incompatible
 	github.com/code-ready/clicumber v0.0.0-20190801094041-02a92c8f7ece
-	github.com/code-ready/goodhosts v0.0.0-20190712111040-f090f3f77c26 // indirect
```

Comment:
???

Reply:
never dare to question these tools, but often it is hard to understand what is happening ...

Reply:
So, i ran `make vendor` because i had to add this new package `golang.org/x/text` and that created this changes,

Reply:
Yes there is unused vendoring. I did the same here: https://github.com/code-ready/crc/pull/1312

Reply:
it's now merged, you can rebase to avoid this change.

Reply:
Sure, will do!